src/scss/_navbar.scss

```diff
@@ -3,7 +3,6 @@
   height: var(--header-height);
   width: 100%;
   padding: 0;
-  overflow: hidden;
```

Comment:
where did this addition come from? we should make sure it wasn't fixing a different bug or something

Reply:
It was added in https://github.com/pydata/pydata-sphinx-theme/pull/286#discussion_r562680610, but I don't directly know why it would have been necessary (I tested a few of the targets that #286 was fixing, and they still seem to be working)